### PR TITLE
Correct issues with newest versions of conda

### PR DESCRIPTION
**Pull request by cvelascof**
Solve https://github.com/conda-tools/conda-execute/issues/50

### CODE REVIEW

File: conda_execute/conda_interface.py [hunk 1]
```diff
@@ -14,7 +14,8 @@ def parse_conda_version_major_minor(string):
 conda_43 = CONDA_VERSION_MAJOR_MINOR >= (4, 3)
 
 from conda.lock import Locked
-from conda.config import user_rc_path, sys_rc_path
+# from conda.config import user_rc_path, sys_rc_path
+from conda.base.context import user_rc_path, sys_rc_path
```
Comment (jakirkham) [inline conda_execute/conda_interface.py]: Could we move this into the `if...else` below?

File: conda_execute/conda_interface.py [hunk 2]
```diff
@@ -14,7 +14,8 @@ def parse_conda_version_major_minor(string):
 conda_43 = CONDA_VERSION_MAJOR_MINOR >= (4, 3)
```
Comment (jakirkham) [inline conda_execute/conda_interface.py]: Also would add `conda_44`.

File: conda_execute/conda_interface.py [hunk 3]
```diff
@@ -24,7 +25,8 @@ def parse_conda_version_major_minor(string):
     from conda.exports import Resolve
     from conda.exports import get_index
 
-    from conda.common.yaml import yaml_load
+    # from conda.common.yaml import yaml_load
+    from conda.common.serialize import yaml_dump, yaml_load
```
Comment (jakirkham) [inline conda_execute/conda_interface.py]: This should also be split across different `conda_4*` branches.

**Comment by jakirkham** [comment]
Thanks @cvelascof.

Couple comments above about `conda` version compatibility above.

Also it would be nice to get `conda` 4.4 and/or 4.5 into the CIs.

**Comment by cvelascof** [comment]
Closing this Pull request. 
Complete solution pulled at https://github.com/conda-tools/conda-execute/pull/52

**Pull request closed without merging**